pkg/topology/checks/component_check.go

```diff
@@ -21,12 +22,16 @@ func ComponentCheckRun() {
 		return
 	}
 	for _, component := range components {
+		jobHistory := models.NewJobHistory("ComponentCheckRelationshipSync", "component", component.ID.String()).Start()
```

Comment:
I don't think this should be per component - it would create a huge number of jobs + at some point we may want to optimize this into a single sql call